### PR TITLE
Formatting, additional info on scope

### PR DESCRIPTION
**Pull request by alterlife**
1. Cleaned up header indentation
2. Added additional info about scope=row and scope=col.
3. Fixed a couple of typos

note: I believe there's also 'colgroup' and 'rowgroup' for complex tables, perhaps that deserves a separate article?

### CODE REVIEW

File: _posts/2016-03-05-accessible-data-tables.md [hunk 1]
```diff
@@ -42,24 +42,27 @@ The semantic purpose of a data table is to present tabular data. Sighted users c
...
+## Row and Column Headings
+For sighted users, row and column headings of tables are often highlighted visually. The screen reader will need this information coded into the markup.
 
-The `scope="row"` is typically put on the first `<th>` of the row. In this case <strong>"January" and "February"</strong> would get them. 
+Column headers should be marked using `scope="col"`. The `scope="col"` typically goes on the `<th>` element. If the `<th>` is a column like the <strong>"Amount Earned", "Amount Spent" and "Amount Saved"</strong> in my example, you would put them all as `scope="col"`. This lets the screen reader know that the `<th>`’s are column headers for that entire column. Column headers should be unique.
+
+Row headers should be marked up using `scope="row"`. The `scope="row"` is typically put on the first `<tr>` of the row. In this case <strong>"January" and "February"</strong> would get them. For optimal usability, the row header selected should be sufficiently unique and readable to identify the row.
```
Comment (svinkle) [inline _posts/2016-03-05-accessible-data-tables.md]: >  the first `<tr>` of the row
should be:
>  the first `<th>` of the row

Comment (alterlife) [inline _posts/2016-03-05-accessible-data-tables.md]: I think it should be `<td>` rather than `<th>` for row headers.

Good catch.

Comment (svinkle) [inline _posts/2016-03-05-accessible-data-tables.md]: Cell header with `scope="row"` would still be a `<th>`.

Comment (alterlife) [inline _posts/2016-03-05-accessible-data-tables.md]: https://www.w3schools.com/tags/att_td_scope.asp

Comment (alterlife) [inline _posts/2016-03-05-accessible-data-tables.md]: That's interesting, The <td> scope attribute is not supported in HTML5. Does that mean only the th is marked with scope and all the rows are automatically read as row headings?

Comment (alterlife) [inline _posts/2016-03-05-accessible-data-tables.md]: I did some more reading, It seems that my original understanding that th should not go into tbody is incorrect, it is recomended and valid for accessibility:
https://stackoverflow.com/questions/5509163/does-html5-ban-th-cells-from-tbody

I see that your example also covers this. Will you be editing the article?

Comment (svinkle) [inline _posts/2016-03-05-accessible-data-tables.md]: Update your PR for now with this change. Going to get more eyes on it before we accept.

Comment (alterlife) [inline _posts/2016-03-05-accessible-data-tables.md]: I've done so.

**Comment by svinkle** [comment]
These changes look good to me.

/cc @Far3 @grayghostvisuals @joe-watkins

**Comment by grayghostvisuals** [comment]
@svinkle Unfortunately I'm not able to donate time to the project right now. Thanks for including me though.

**Pull request merged**